src/software/ai/passing/pass_generator_test.cpp

```diff
@@ -219,7 +188,7 @@ TEST_F(PassGeneratorTest, check_pass_does_not_converge_to_self_pass)
     UNUSED(converged_score);
 }
 
-TEST_F(PassGeneratorTest, test_passer_point_changes_are_respected)
+TEST_F(PassGeneratorTest, DISABLED_test_passer_point_changes_are_respected)
```

Comment:
why disabled?

Reply:
ah that slipped through, re-enabled 2 and delete 1 of them because its no-longer relevant